include/abstract_index.h

```diff
@@ -79,6 +79,9 @@ class AbstractIndex
                                                       const size_t K, const uint32_t L, IndexType *indices,
                                                       float *distances);
 
+    template <typename data_type, typename tag_type, typename label_type>
+    int insert_point(const data_type *point, const tag_type tag, const std::vector<label_type> &labels = {});
```

Comment:
Could we document why there are multiple "insert_point" functions and the scenarios that require them?

Reply:
Sure, I would add this to appropriate Readme file.

Reply:
Added docs to dynamic_index.md file, also added instructions to build and search dynamic filtered index.

Reply:
Added instructions in dynamic_index.md for filtered streaming index.